windows/packaging/exe/make_config.yaml

```diff
@@ -0,0 +1,10 @@
+# The value of AppId uniquely identifies this application.
+# Do not use the same AppId value in installers for other applications.
+app_id: 4FB521BA-98C0-43D0-8B74-891EA6D7B2E5 # TODO!: Change this!

```

Comment:
You should change this

Reply:
There is a merge problem here, I'll fix it this is not made by me.